package/MDAnalysis/coordinates/PDB.py

```diff
@@ -391,14 +391,21 @@ def _read_frame(self, frame):
             elif line[:6] == 'CRYST1':
                 # does an implicit str -> float conversion
                 try:
-                    self.ts._unitcell[:] = [line[6:15], line[15:24],
-                                            line[24:33], line[33:40],
-                                            line[40:47], line[47:54]]
+                    cell_dims = [line[6:15], line[15:24],
+                                  line[24:33], line[33:40],
+                                  line[40:47], line[47:54]]  
                 except ValueError:
                     warnings.warn("Failed to read CRYST1 record, "
                                   "possibly invalid PDB file, got:\n{}"
                                   "".format(line))
-
+                else:
+                    if (np.array(cell_dims, dtype=float) == 
+                        np.array([1, 1, 1, 90, 90, 90], dtype=float)).all():
+                        warnings.warn("1 A^3 CRYST1 record," 
+                                      " possibly an EM structure file",stacklevel=2)
```

Comment:
I'm not sure that setting stacklevel to 2 here is appropriate. If I remember how this works properly, this should make the warning trace back to `_read_next_timestep` instead. It probably is best to leave the warning pointing at this exact part of the code so that it's easier to understand why this warning is occuring (see above warning).